src/lib/sql/sql_translator.cpp

```diff
@@ -603,6 +608,7 @@ SQLTranslator::TableSourceState SQLTranslator::_translate_table_origin(const hsq
 
       } else if (Hyrise::get().storage_manager.has_view(hsql_table_ref.name)) {
         const auto view = Hyrise::get().storage_manager.get_view(hsql_table_ref.name);
+        _cacheable = false;
```

Comment:
Not quite sure what made this change necessary, but whatever it is should be tested.